build.gradle

```diff
@@ -71,7 +71,12 @@ subprojects {
         }
 
         gradleTest {
-            versions '3.0', '3.5', '4.1', '4.7', '4.9'
+            // https://github.com/jruby-gradle/jruby-gradle-plugin/issues/346
+            if (JavaVersion.current().java11) {
+                versions '5.0'
```

Comment:
I would change the JDK11 setup to be `versions '5.0', '5.4.1'` and the JDK8one  to be `versions '4.2','4.10.2', '5.3.1'`. IN that way we'll get better coverage.